src/main/java/org/flockofseagles/UI/DocumentController.java

```diff
@@ -0,0 +1,21 @@
+package org.flockofseagles.UI;
+
+import javafx.fxml.Initializable;
+
+import java.awt.*;
+import java.net.URL;
+import java.util.ResourceBundle;
+
+public class DocumentController extends Dialog implements Initializable
+{
+	public DocumentController(Frame owner)
+	{
+		super(owner);
+	}
+
+	@Override
+	public void initialize(URL location, ResourceBundle resources)
+	{
```

Comment:
I can't remember if I sent over the IntelliJ code style config before, but this doesn't match the desired brace style for our team's coding standards

Reply:
Fixed in 12eca2b